ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -50,9 +77,7 @@ case class CommandSubmitter(
       )
     } yield {
       logger.info("Commands submitted successfully.")
-      CommandSubmitter.SubmissionSummary(
-        observers = observers
-      )
+      CommandSubmitter.SubmissionSummary(observers = observers)
```

Comment:
Using `SubmissionSummary` with wrapped observers that are passed as a method argument doesn't make sense here. Observers are known beforehand so we may return something like `case object SubmissionSuccess`.
But there's going to be a cleanup of the bench-tool, so we can add a TODO note and leave it for now.

Reply:
Added a TODO 👍